### PR TITLE
Content sweep of build 1

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Committing this before lunch – there'll be lots more to come :(

### Context

Why are you making this change? What might surprise someone about it?

### Changes proposed in this pull request

If there are UI changes, please include Before and After screenshots.

### Guidance to review

How could someone else check this work? Which parts do you want more feedback on?

### Link to Trello card

[123 - Example Trello card](http://trello.com/123-example-card)

### Env vars

- [ ] If this PR introduces new environment variables, they have been added to all necessary parts of the Azure config based on the [documentation in the README](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/candidate_interface/start_page/show.html.erb [hunk 1]
```diff
@@ -21,9 +21,12 @@
       <li>grade 4 (C) or above in GCSE science if you want to teach primary</li>
     </ul>
     <p class="govuk-body">
-      Before you start a teacher training course you’ll also need to confirm you have the health and physical capacity to start training.
+      Before you start a teacher training course, your training provider will usually send you a health questionnaire to complete. This confirms you have the fitness and physical capacity to start training.
     </p>
-    <h2 class="govuk-heading-m">If you don’t have a degree</h2>
+    <h2 class="govuk-heading-m">If you have a disability</h2>
+      <p class="govuk-body">It is against the law for training providers to discriminate against teacher training candidates with disabilities or special educational needs. <a href="https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability">Learn more</a>.</p>
```
Comment (chubberlisk) [inline app/views/candidate_interface/start_page/show.html.erb]: This link needs to be a `govuk-link`. What we can do instead is this using a lil helper we have created:

```suggestion
      <p class="govuk-body">It is against the law for training providers to discriminate against teacher training candidates with disabilities or special educational needs. <%= govuk_link_to 'Learn more', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```

**Comment by tvararu** [comment]
Rebased this for you @laurahermionetennant92, feel free to merge when it's green!

File: app/views/candidate_interface/start_page/show.html.erb [hunk 2]
```diff
@@ -21,9 +21,12 @@
       <li>grade 4 (C) or above in GCSE science if you want to teach primary</li>
     </ul>
     <p class="govuk-body">
-      Before you start a teacher training course you’ll also need to confirm you have the health and physical capacity to start training.
+      Before you start a teacher training course, your training provider will usually send you a health questionnaire to complete. This confirms you have the fitness and physical capacity to start training.
     </p>
-    <h2 class="govuk-heading-m">If you don’t have a degree</h2>
+    <h2 class="govuk-heading-m">If you have a disability</h2>
+      <p class="govuk-body">It is against the law for training providers to discriminate against teacher training candidates with disabilities or special educational needs. <%= govuk_link_to 'Learn more', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```
Comment (fofr) [inline app/views/candidate_interface/start_page/show.html.erb]: This link isn't accessible.

File: app/views/content/terms_candidate.md [hunk 3]
```diff
@@ -7,8 +7,7 @@ Using Apply for teacher training means you agree to these terms of use.
 
 ### Stage 1: submitting your initial application(s)
 
-You can apply for up to 3 courses through Apply for teacher training at this
-stage. You can also apply to 3 courses through UCAS if you want to.
+During the first round of your application, you should apply to no more than three courses in total, whether you use Apply for teacher training, UCAS Teacher Training, or a combination of the two.
```
Comment (fofr) [inline app/views/content/terms_candidate.md]: `than 3 courses in total` – we should stick to the GOV.UK style: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#numbers

**Pull request merged**